test/io/ferguson/ctests/qio_formatted_test.compopts

```diff
@@ -1 +1 @@
--DCHPL_RT_UNIT_TEST  $CHPL_HOME/runtime/src/qio/qio_formatted.c $CHPL_HOME/runtime/src/qio/qio.c $CHPL_HOME/runtime/src/qio/qbuffer.c $CHPL_HOME/runtime/src/qio/sys.c $CHPL_HOME/runtime/src/qio/sys_xsi_strerror_r.c $CHPL_HOME/runtime/src/qio/qio_error.c $CHPL_HOME/runtime/src/qio/deque.c -lpthread
+-DCHPL_RT_UNIT_TEST  $CHPL_HOME/runtime/src/qio/qio_formatted.c $CHPL_HOME/runtime/src/qio/qio.c $CHPL_HOME/runtime/src/qio/qbuffer.c $CHPL_HOME/runtime/src/qio/sys.c $CHPL_HOME/runtime/src/qio/sys_xsi_strerror_r.c $CHPL_HOME/runtime/src/qio/qio_error.c $CHPL_HOME/runtime/src/qio/deque.c -lpthread -lm
```

Comment:
I'll call out in the PR message why this change was necessary, just for future reference